libp2pdht/private/eth/p2p/discoveryv5/messages_encoding.nim

```diff
@@ -117,6 +127,7 @@ proc decodeMessage*(body: openArray[byte]): DecodeResult[Message] =
       of ping: rlp.decode(message.ping)
       of pong: rlp.decode(message.pong)
       of findNode: rlp.decode(message.findNode)
+      of findNodeFast: rlp.decode(message.findNodeFast)
```

Comment:
What do you think about adding in Protobuf en/decoding instead of RLP since we'll be eliminating all RLP en/decoding soon?

Reply:
This would be point 2 of https://github.com/status-im/nim-libp2p-dht/issues/3, right?